firefox-ios/Client/Frontend/Summary/LiteLLMClient.swift

```diff
@@ -7,7 +7,7 @@ import Common
 import Shared
 
 /// A lightweight client for interacting with a chat completions endpoint.
-public class LiteLLMClient: NSObject, URLSessionDataDelegate {
+public final class LiteLLMClient: NSObject, URLSessionDataDelegate, @unchecked Sendable {
```

Comment:
If we are making this `@unchecked Sendable`, it means we tell the compiler "This isn't thread-safe, but I am myself making this thread-safe due to internal locking mechanisms so you can trust me Mister Compiler". For the sake of the migration we can mark those as `@unchecked Sendable` without the locking mechanisms, but we need to create follow up tickets to tackle this in [Phase 2 of the migration](https://mozilla-hub.atlassian.net/browse/FXIOS-12669). Could you create a ticket and add it as a TODO comment above this class?